dirsrvtests/conftest.py

```diff
@@ -115,7 +115,7 @@ def pytest_runtest_makereport(item, call):
                     text = asan_report.read()
                     extra.append(pytest_html.extras.text(text, name=os.path.basename(f)))
             for f in glob.glob(f'{p.log_dir.split("/slapd",1)[0]}/*/*'):
-                if 'rotationinfo' not in f:
+                if 'rotationinfo' not in f and not f.endswith('.gz'):
```

Comment:
IMHO we should handle compressed log too here rather than just ignoring it:
-                if 'rotationinfo' not in f:
+                if f.endswith('gz'):
+                    with subprocess.Popen(["/usr/bin/zcat", f], stdout=PIPE) as dirsrv_log:
+                        text = dirsrv_log.read()
+                        log_name = os.path.basename(f)
+                        instance_name = os.path.basename(os.path.dirname(f)).split("slapd-",1)[1]
+                        extra.append(pytest_html.extras.text(text, name=f"{instance_name}-{log_name}"))
+                elif 'rotationinfo' not in f:

Reply:
I fixed this by using python's gzip command instead (but it's basically the same thing) :-)